packages/@lwc/compiler/README.md

```diff
@@ -43,7 +43,6 @@ const { code } = transformSync(source, filename, options);
     -   `namespace` (type: `string`, required) - namespace of the component, e.g. `x` in `x/foo`.
     -   `stylesheetConfig` (type: `object`, default: `{}`) - The stylesheet compiler configuration to pass to the `@lwc/style-compiler`.
     -   `experimentalDynamicComponent` (type: `DynamicImportConfig`, default: `null`) - The configuration to pass to `@lwc/compiler`.
-    -   `experimentalDynamicDirective` (type: `boolean`, default: `false`) - The configuration to pass to `@lwc/template-compiler` to enable deprecated dynamic components.
```

Comment:
Sounds like a plan!

Reply:
@nolanlawson now that I think about it, the `experimentalDynamicDirective` option was released at the start of 246 and I think the only consumers are `lwc-platform`.  

I think the `experimentalDynamicComponent` may be the one that affects more downstream dependencies. 

I'll double-check to see if anyone else is using `experimentalDynamicDirective` but otherwise I think it should be safe to remove that option.

Reply:
@jmsjtu Should this be green in Nucleus before merging?